utils/fragment_range.hh

```diff
@@ -198,3 +198,16 @@ decltype(auto) with_linearized(const View& v, Function&& fn)
         return fn(linearized(v));
     }
 }
+
+class single_fragmented_view {
+    bytes_view _view;
+public:
+    using fragment_type = bytes_view;
+    explicit single_fragmented_view(bytes_view bv) : _view(bv) {}
```

Comment:
I made it explicit specifically so that it isn't seamless. It will make it easier to find potential linearizations and/or uses of old `bytes_view` versions of stuff.